services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/KibanaHelper.groovy

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.functionaltests.helpers
+
+import groovy.util.logging.Slf4j
+import org.openkilda.testing.service.elastic.ElasticHelper
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import groovy.json.JsonBuilder
+import org.springframework.stereotype.Component
+
+/**
+ * A set of helper methods to access, read and filter log entries from the log management system
+ */
+
+@Component
+@Slf4j
+class KibanaHelper {
```

Comment:
Did you investigate official clients?
https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/java-search.html
https://www.elastic.co/blog/found-java-clients-for-elasticsearch
https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/java-query-dsl.html

If they are usable, we should be definitely using them.

Reply:
Unfortunately, wiring up a full-fledged elastic client turned out to be way too complicated for our simple tasks (and it'll require users to get into ES details, which are not pretty)